src/opera/pge/dswx_s1/schema/dswx_s1_sas_schema.yaml

```diff
@@ -65,4 +65,4 @@ runconfig:
 
             product_version: num(required=False)
 
-        log_file: 
\ No newline at end of file
+        log_file: str()
```

Comment:
We need to keep this file in sync with the version ADT maintains, so this should be set to `str(required=False)`